### ISSUE TITLE
simprod-scripts fullSimulation.py (Trac #887)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#881]
Clean up fullSimulation.py to reflect what we actually do in production. Specifically:

* remove baddomlist tweaking
* consider using the event generation times we use in production? (do these need to even be specified?)
* remove comments (or anything else) referencing claudio
* use cvmfs spline table paths
* add support for segments.MultiDomEffSample

Also, consider turning calibration off by default, since this is in the simulation meta-project and can't work there.

Migrated from https://code.icecube.wisc.edu/ticket/887
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "Clean up fullSimulation.py to reflect what we actually do in production. Specifically:\n\n* remove baddomlist tweaking\n* consider using the event generation times we use in production? (do these need to even be specified?)\n* remove comments (or anything else) referencing claudio\n* use cvmfs spline table paths\n* add support for segments.MultiDomEffSample\n\nAlso, consider turning calibration off by default, since this is in the simulation meta-project and can't work there.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "simprod-scripts fullSimulation.py", 
    "priority": "normal", 
    "keywords": "simprod-scripts", 
    "time": "2015-02-25T01:56:06", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150311T04:07:33: 
* olivas changed owner from "" to "david.schultz"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150709T22:11:52: david.schultz changed milestone from "" to "IceSim 4.2"

**Comment by icecube-trac** [comment]
Trac update at 20150826T16:20:32: david.schultz commented:

> depends on #1288. waiting for that ticket to clear.

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:40:00: olivas changed milestone from "IceSim 4.2" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20160120T22:20:12: 
* david.schultz changed status from "assigned" to "closed"
* david.schultz commented:

> In changeset 141085:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141085"
> remove I3EventHeader from generators (done in triggersim now). run all the tests we have. fix up fullSimulation to remove claudio-specific things, calibration, etc. closes #881
> ```


* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas commented:

> Milestone IceSim 5.0 deleted


* olivas changed milestone from "IceSim 5.0" to ""
